apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/FissClaimTransformerV2.java

```diff
@@ -123,15 +131,45 @@ private static Claim transformClaim(RdaFissClaim claimGroup, boolean includeTaxN
    */
   private static List<Claim.SupportingInformationComponent> getSupportingInfo(
       RdaFissClaim claimGroup) {
-    return Strings.isBlank(claimGroup.getFreqCd())
-        ? null
-        : List.of(
-            new Claim.SupportingInformationComponent()
-                .setCategory(createCodeableConcept(C4BBSupportingInfoType.TYPE_OF_BILL))
-                .setCode(
-                    new CodeableConcept(
-                        new Coding(BBCodingSystems.FISS.FREQ_CD, claimGroup.getFreqCd(), null)))
-                .setSequence(1));
+    List<Claim.SupportingInformationComponent> supportingInfo = new ArrayList<>();
+    int sequenceNumber = 1;
+
+    if (Strings.isNotBlank(claimGroup.getFreqCd())) {
+      supportingInfo.add(
+          new Claim.SupportingInformationComponent()
+              .setSequence(sequenceNumber++)
```

Comment:
I'm not sure what the intent was with this, but since the way incrementing and assignment is a bit tricky to intuitively understand (see: https://stackoverflow.com/questions/24564603/java-increment-and-assignment-operator ) I'd reccomend adjusting the way this is incremented to be more intentful. (Specifically increment it separately before or after the assignment, as desired.)

Reply:
I've always considered increment operators to be rather fundamental knowledge, present in most of the common place languages of today, and among basic skills one would pick up early on.  If this is a style that is less preferred in BFD though, I could make that adjustment.